gr-network/python/network/qa_tcp_sink.py

```diff
@@ -31,6 +31,7 @@ def tearDown(self):
 
     def test_restart(self):
         serversocket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        serversocket.settimeout(30.0)
```

Comment:
I used a large value (30 seconds) here to make sure that the test doesn't become flaky when running on a heavily loaded machine. The test still completes immediately when it passes, and the 30-second delay will only occur if the test fails.